Makefile

```diff
@@ -132,6 +132,10 @@ anvil_start_with_block_time:
 	@echo "Starting Anvil..."
 	anvil --load-state contracts/scripts/anvil/state/alignedlayer-deployed-anvil-state.json --block-time 7
 
+anvil_start_with_block_time_with_more_prefunded:
+	@echo "Starting Anvil..."
+	anvil --load-state contracts/scripts/anvil/state/alignedlayer-deployed-anvil-state.json --block-time 7 -a 2000
```

Comment:
This looks like what we really want is to parameterize how much funding we want, while giving a sane default. Otherwise, we may later need a target `anvil_start_with_block_time_with_even_more_prefunded` and it'd be a never-ending story.

Reply:
more prefunded is more prefunded accounts! not quantity of funds, i will add the `accounts` for more clarity